### DISCUSSION TITLE
Regarding gp.plot_3d failure

### DISCUSSION THREAD

**Discussion by thelearningcurves** [gempy-project/gempy#788] [Q&A]
I need to make layered sections for gravity modelling but when I try to render 3d display using the gp.plot_3d function, jupyter notebook crashes. In the terminal, the It says " Cannot create GLX context". This is really frustrating. So, I wanted to ask if I could import the model as a VTU or VTK file compatible with ParaView? Or if anyone understands my problem, please help,
I am running Ubuntu 22.0 on WSL2 system.

**Comment by javoha** [comment]
Hello @thelearningcurves, 
as this seems to be regarding an ongoing issue ( #785 ), I will close this discussion. Please refrain from opening multiple issues/discussions on the same topic and provide answers to my follow-up questions in the issue. I will also try answer the vtk export question there.

**Comment by thelearningcurves** [comment]
Thanks for reminding me. I will take that into due consideration next time
